library/cartridge_edit_topology.py

```diff
@@ -489,6 +499,9 @@ def get_server_params(instance_name, new_instance, old_instances, allow_missed_i
     if new_instance.get('expelled') is True:
         server_params['expelled'] = True
     else:
+        for flag_name in ['disabled']:
```

Comment:
There is no need to add other flags for such a server, let's continue.

Reply:
No, this is done to ensure that the instance is always up to date. So, if suddenly it is turned on through the UI, nothing would break